enterprise-suite/console-operator/deploy/crds/console_v1alpha1_console_cr.yaml

```diff
@@ -16,13 +16,15 @@ spec:
   # Update these versions for a release.
   #
   # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
-  esConsoleVersion: v1.2.5
+  esConsoleVersion: v1.2.6
   # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/console-api
-  esMonitorVersion: v1.2.2
+  esMonitorVersion: v1.2.3
```

Comment:
Isn't there something that checks these are up to date? How come we're changing them in this PR?

Reply:
Yes, `verify-operator` build step checks that operator resources are in-sync with helm charts.